hello.py

```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env python
 
-print("Hello, World!")
+for i in range(0,10):
+    if (i % 3):
```

Comment:
You want to check if `i` is divisible by 3 and this you can do with `i % 3 == 0` (0 is coerced to False). Does this make sense to you?